bytes/src/main/java/net/consensys/cava/bytes/MutableBytes.java

```diff
@@ -212,6 +212,42 @@ static MutableBytes wrapByteBuffer(ByteBuffer byteBuffer, int offset, int size)
    */
   void set(int i, byte b);
 
+  /**
+   * Increments the value of the bytes by 1, treating the value as big endian.
+   *
+   * If incrementing overflows the value, all bits flip, ie incrementing 0xFFFF will return 0x0000.
+   */
+  default MutableBytes increment() {
```

Comment:
New methods in `Bytes` should also be added to `DelegatingBytes` et al so that specializations are accessible. Also, consider adding a specialization for the array version.